src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -379,44 +379,73 @@ public void Clear()
         cachedStorageScores.Clear();
     }
 
-    public (float PilusScore, float OxytoxyScore, float MucilageScore) GetPredationToolsRawScores(
+    public (float PilusScore, float OxytoxyScore, float SlimeJetScore) GetPredationToolsRawScores(
         MicrobeSpecies microbeSpecies)
     {
         if (cachedPredationToolsRawScores.TryGetValue(microbeSpecies, out var cached))
             return cached;
 
         var pilusScore = 0.0f;
         var oxytoxyScore = 0.0f;
-        var mucilageScore = 0.0f;
+        var slimeJetScore = 0.0f;
 
         var organelles = microbeSpecies.Organelles.Organelles;
         var organelleCount = organelles.Count;
+        var slimeJetsCount = 0;
+        var topPosition = 0;
+        var bottomPosition = 0;
+        List<int> slimeJetsHorizontalPositions = new List<int>();
```

Comment:
is this named correctly? Based on the code below it seems like these are vertical positions.